src/neat/neat_functions.py

```diff
@@ -92,3 +92,83 @@ def delta(genome1: Genome, genome2: Genome, c1: float = 1.0, c2: float = 1.0, c3
     delta = c1 * (excess / n) + c2 * (disjoint / n) + c3 * (total_diff / matching)
 
     return delta
+
+
+###__Weight Mutation__###
+
+
+def mutateWeights(genes: List):
+	"""
+	Function to randomly mutates the genome to alter the connection weights
+	:param genes: List of genes
+	"""
+	
+	for i, connection in enumerate(genes):
+		if random() < 0.75:
```

Comment:
Paper has this as .8 chance of weights being perturbed and then .9 it's randomly perturbed and .1 it's assigned a random value

Reply:
Also should use something smaller than -1, 1. Maybe -.05, .05 (but we can test this for what works best once it's all working